### PR TITLE
I18n sync HOC 11/20

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: i18n/locales/it-IT/hourofcode/how-to/index.md [hunk 1]
```diff
@@ -13,13 +13,11 @@ Unisciti al movimento ed introduci un gruppo di studenti alla loro prima ora di
...
 
-Noi forniamo un'ampia scelta di divertenti [esercitazioni autogestite](%= resolve_url('/learn') %) ed altrettante [gestite dall'insegnante](%= resolve_url('https://code.org/educate/teacher-led') %) per partecipanti di tutte le età, create da diversi partner.
-
-[![](/images/fit-700/tutorials.png)](%=resolve_url('/learn') %)
+We provide a variety of fun, [student-guided tutorials](%= resolve_url('/learn') %) for all age groups and experience levels. Students do the activities on their own, though many activities include lesson plans for teachers (you'll see the link when you click the activity) to guide discussion or extend the activity. [![](/images/fit-700/tutorials.png)](%=resolve_url('/learn') %)
 
 ## 3. Promuovi la tua Ora del Codice
 
-Promuovi la tua Ora del Codice [con questi strumenti](%= resolve_url('/promote/resources') %) e incoraggia altri ad ospitare i loro eventi.
+Promote your Hour of Code [with these tools](%= resolve_url('/promote/resources') %) and encourage others to host their own events.
```
Comment (tanyaparker) [inline i18n/locales/it-IT/hourofcode/how-to/index.md:20]: Looks like another example where a source string changed early in the file and we lost translations later in the file. 🤷‍♀️ 

I re-committed the translations in crowdin.

File: i18n/locales/zh-CN/hourofcode/how-to/index.md [hunk 2]
```diff
@@ -33,34 +31,34 @@ nav: how_to_nav
 
 <img src="/images/fit-350/group_ipad.jpg" />
 
-## 5. Start your Hour of Code off with an inspiring speaker or video
+## 5）以一段激励演讲或视频来开始编程一小时活动
```
Comment (tanyaparker) [inline i18n/locales/zh-CN/hourofcode/how-to/index.md:34]: Interestingly enough this file lost some translations but added some too. So confused.

Comment (Hamms) [reply i18n/locales/zh-CN/hourofcode/how-to/index.md:34]: Looks like there was generally some weirdness with Chinese; I re-synced after fixing some syntax issues, and a whole bunch of stuff changed. PTAL

**Comment by tanyaparker** [comment]
Awesome, Chinese looks better. But don't know if this has happened in other languages? I'm guessing the tests will catch?

**Comment by Hamms** [comment]
Tests will catch syntax errors, but not lost translations.

I can do another full re-sync after I'm done with the code.org i18n sync

**Comment by tanyaparker** [comment]
Sure. It can be the 27th sync.

**Pull request merged**